A32-xds-circuit-breaking.md

```diff
@@ -63,14 +55,28 @@ limits to very high values, for example, to
 
 ## Detailed design
 
-Each CDS LB policy maintains a configured limit for the maximum number of 
+With the introduction of [gRFC A28: xDS Traffic Splitting and Routing](https://github.com/grpc/proposal/blob/master/A28-xds-traffic-splitting-and-routing.md), 
+it became possible for there to be more than one CDS LB policy instance for a 
+given cluster (e.g., if one route pointed to a given cluster and a different 
+route split traffic between that cluster and other clusters).  However, after 
+implementing the changes described in [gRPC A31: xDS RouteActions Support](https://github.com/grpc/proposal/pull/192),
+there will be only one CDS LB policy instance for a given cluster at any time
+in the LB policy tree. Since each CDS LB policy creates exactly one child EDS
+LB policy for endpoint discovery, both of them are part of the load balancing
+logic for requests sent to the specific cluster. Circuit breakers can be 
+implemented in either policies. In order to easily record and report requests 
```

Comment:
We should probably explicitly say that the reason for doing this in the EDS policy is that that's where we're already handling the drops requested by EDS, so it makes sense to handle these drops in the same place.

Reply:
Updated.